README.md

```diff
@@ -5,4 +5,38 @@
 [![Quality Gate](https://sonarcloud.io/api/project_badges/measure?project=com.powsybl%3Apowsybl-gse&metric=alert_status)](https://sonarcloud.io/dashboard?id=com.powsybl%3Apowsybl-gse)
 [![MPL-2.0 License](https://img.shields.io/badge/license-MPL_2.0-blue.svg)](https://www.mozilla.org/en-US/MPL/2.0/)
 
-http://www.powsybl.com
+Powsybl (Power System Blocks) is an open source framework written in Java, that makes it easy to write complex software for power systems’ simulations and analysis.
+
+![PowSyBl Logo](https://raw.githubusercontent.com/powsybl/powsybl-gse/master/gse-spi/src/main/resources/images/logo_lfe_powsybl.svg?sanitize=true)
+
+Read more at http://www.powsybl.com !
+
+This project and everyone participating in it is governed by the [PowSyBl Code of Conduct](https://github.com/powsybl/.github/blob/master/CODE_OF_CONDUCT.md). By participating, you are expected to uphold this code. Please report unacceptable behavior to [powsybl.ddl@rte-france.com](mailto:powsybl.ddl@rte-france.com).
+
+## Project Structure
+
+The powsybl-gse project provides JavaFX components, a base JavaFX application and a build system to create standalone distributable GUI applications exposing functionality from the PowSyBl framework.
+
+![GSE Demo](https://user-images.githubusercontent.com/89208/52073009-b55b0f80-2586-11e9-8c7d-40f7abfb0ff6.gif)
+
+## Environment requirements
+
+  * JDK *(1.8 or greater)* with JavaFX (e.g. the oracle JDK 1.8)
+  * Maven *(3.3.9 or greater)*
+  * powsybl-core : https://github.com/powsybl/powsybl-core
```

Comment:
If you are an end-user, you don't need to download/build powsybl-core: dependencies will be downloaded by maven central. `powsybl-core` is needed if you build the GSE from sources.
Maybe we should have a `Getting started` section here too, to redirect people to the web site ([itools](https://powsybl.github.io/docs/installation/itools-packager.html) or [javaFX](https://powsybl.github.io/docs/installation/javafx-packager.html))